server/src/service.rs

```diff
@@ -292,6 +292,13 @@ impl Service {
         flags: BitFlags<zbus::fdo::RequestNameFlags>,
     ) -> Result<(), Error> {
         let connection = zbus::Connection::session().await?;
+
+        if let Some(unique_name) = connection.unique_name() {
+            tracing::info!("Got unique name {unique_name} on the bus");
+        }
+
+        setup_dbus_proxy(&connection).await?;
```

Comment:
Maybe spawn the task here to avoid two functions instead of one? Also the name is confusing

Reply:
Renamed both fns.